tests/integration/test_backends.py

```diff
@@ -186,3 +191,44 @@ def test_runtime_backend_use_statements(ws):
 """
     result = commands.run(permission_denied_query)
     assert result == "PASSED"
+
+
+def wait_until_seconds_rollover(*, rollover_seconds: int = 10) -> None:
+    """Wait until the next rollover.
+
+    Useful to align concurrent writes.
+
+    Args:
+        rollover_seconds (int) : The multiple of seconds to wait until the next rollover.
+    """
+    nanoseconds = 1e9
+    microseconds = 1e6
+
+    now = time.clock_gettime_ns(time.CLOCK_REALTIME)
+    target = math.ceil(now / nanoseconds // rollover_seconds) * nanoseconds * rollover_seconds
+
+    # To hit the rollover more accurate, first sleep until almost target
+    nanoseconds_until_almost_target = (target - now) - microseconds
+    time.sleep(max(nanoseconds_until_almost_target / 1e9, 0))
+
+    # Then busy-wait until the rollover occurs
+    while time.clock_gettime_ns(time.CLOCK_REALTIME) < target:
+        pass
+
+
+def test_runtime_backend_handles_concurrent_append(sql_backend, make_random) -> None:
+    table_name = f"lsql_test_{make_random(8)}"
+    sql_backend.execute(f"CREATE TABLE IF NOT EXISTS {table_name} (x int, y float)")
+    sql_backend.execute(f"INSERT INTO {table_name} BY NAME SELECT r.id AS x, random() AS y FROM range(100000000) r")
```

Comment:
I know where this came from, and it was a little… extreme.
```suggestion
    sql_backend.execute(f"CREATE TABLE {table_name} (x int, y float) AS (SELECT r.id AS x, random() AS y FROM range(1000000) r")
```